src/output-json-alert.c

```diff
@@ -97,6 +97,8 @@
             LOG_JSON_APP_LAYER  |                                  \
             LOG_JSON_RULE_METADATA)
 
+#define _JSON_BODY_LOGGING  (LOG_JSON_HTTP_BODY | LOG_JSON_HTTP_BODY_BASE64)
```

Comment:
why is there a leading underscore?

Reply:
Just to indicate it's private; however, it's private anyway since it's not in a header file.

I'll remove the leading `_`.